lua/methods/group.lua

```diff
@@ -0,0 +1,24 @@
+--
+-- RPC unit actions
+-- https://wiki.hoggitworld.com/view/DCS_Class_Group
+--
+
+local GRPC = GRPC
+
+GRPC.methods.getUnits = function(params)
+  -- https://wiki.hoggitworld.com/view/DCS_func_getByName
+  local group = Group.getByName(params.groupName)
+  if group == nil then
+    return GRPC.errorNotFound("group does not exist")
+  end
+
```

Comment:
Just want to confirm. There is a race condition here if the group gets removed before we call `getUnits`.

I am personally fine with it since I think the changes of that actually happening are infintessimal. Just want to make sure you are also happy with it.

Reply:
Good question, I am somehow in the assumption that the lua blocks the whole mission simulation so that this case would be impossible. But I am just assuming, I don't know if it is really this case